jscripts/jquery.plugins.min.js

```diff
@@ -1 +1,67 @@
te").find("."+d.prefix+"arrow").hide(),n.jqi.find("."+d.prefix+"state:visible").addClass(d.prefix+"parentstate"),a.addClass(d.prefix+"substate")):n.jqi.find("."+d.prefix+"state").not(a).slideUp(s.promptspeed).find("."+d.prefix+"arrow").hide(),n.currentStateName=r.name,a.slideDown(s.promptspeed,function(){var i=t(this);n.enableStateButtons(),"string"==typeof r.focus?i.find(r.focus).eq(0).focus():i.find("."+d.prefix+"defaultbutton").focus(),i.find("."+d.prefix+"arrow").show(s.promptspeed),"function"==typeof o&&n.jqib.on("impromptu:statechanged",o),n.jqib.trigger("impromptu:statechanged",[e]),"function"==typeof o&&n.jqib.off("impromptu:statechanged",o)}),i||n.position())}return a},nextState:function(t){var e=this,i=e.getCurrentState().next();return i.length>0&&e.goToState(i.data("jqi-name"),t),i},prevState:function(t){var e=this,i=e.getCurrentState().prev();return i.length>0&&e.goToState(i.data("jqi-name"),t),i}},t.prompt=function(t,i){return new e(t,i).jqi},t.each(e,function(e,i){t.prompt[e]=i}),t.each(e.prototype,function(i,o){t.prompt[i]=function(){var t=e.getLast();if(t&&"function"==typeof t[i])return t[i].apply(t,arguments)}}),t.fn.prompt=function(e){void 0===e&&(e={}),void 0===e.withDataAndEvents&&(e.withDataAndEvents=!1),t.prompt(t(this).clone(e.withDataAndEvents).html(),e)},window.Impromptu=e}),function(t){var e=!1;if("function"==typeof define&&define.amd&&(define(t),e=!0),"object"==typeof exports&&(module.exports=t(),e=!0),!e){var i=window.Cookies,o=window.Cookies=t();o.noConflict=function(){return window.Cookies=i,o}}}(function(){function t(){for(var t=0,e={};t<arguments.length;t++){var i=arguments[t];for(var o in i)e[o]=i[o]}return e}function e(i){function o(e,n,s){var a;if("undefined"!=typeof document){if(arguments.length>1){if("number"==typeof(s=t({path:"/"},o.defaults,s)).expires){var r=new Date;r.setMilliseconds(r.getMilliseconds()+864e5*s.expires),s.expires=r}s.expires=s.expires?s.expires.toUTCString():"";try{a=JSON.stringify(n),/^[\{\[]/.test(a)&&(n=a)}catch(t){}n=i.write?i.write(n,e):encodeURIComponent(String(n)).replace(/%(23|24|26|2B|3A|3C|3E|3D|2F|3F|40|5B|5D|5E|60|7B|7D|7C)/g,decodeURIComponent),e=(e=(e=encodeURIComponent(String(e))).replace(/%(23|24|26|2B|5E|60|7C)/g,decodeURIComponent)).replace(/[\(\)]/g,escape);var l="";for(var d in s)s[d]&&(l+="; "+d,!0!==s[d]&&(l+="="+s[d]));return document.cookie=e+"="+n+l}e||(a={});for(var c=document.cookie?document.cookie.split("; "):[],u=/(%[0-9A-Z]{2})+/g,p=0;p<c.length;p++){var f=c[p].split("="),h=f.slice(1).join("=");'"'===h.charAt(0)&&(h=h.slice(1,-1));try{var m=f[0].replace(u,decodeURIComponent);if(h=i.read?i.read(h,m):i(h,m)||h.replace(u,decodeURIComponent),this.json)try{h=JSON.parse(h)}catch(t){}if(e===m){a=h;break}e||(a[m]=h)}catch(t){}}return a}}return o.set=o,o.get=function(t){return o.call(o,t)},o.getJSON=function(){return o.apply({json:!0},[].slice.call(arguments))},o.defaults={},o.remove=function(e,i){o(e,"",t(i,{expires:-1}))},o.withConverter=e,o}return e(function(){})});
```

Comment:
I figure this could be minified further, but what tool to use for it?

Reply:
[`uglifyjs`](https://github.com/mishoo/UglifyJS2) is usually my choice.